python/_eos.cc

```diff
@@ -673,6 +673,7 @@ BOOST_PYTHON_MODULE(_eos)
             Represents the log(likelihood) of a Bayesian analysis undertaken with the :class:`Analysis <eos.Analysis>` class.
         )", init<Parameters>())
         .def("add", (void (LogLikelihood::*)(const Constraint &)) &LogLikelihood::add)
+        .def("add", (void (LogLikelihood::*)(const LogLikelihoodBlockPtr &)) &LogLikelihood::add)
```

Comment:
Uhh, does this work? C++ supports overloading. Does Boost-Python support overloading C++ functions? Checking....

Reply:
Nevermind, this is exactly as it is intended. See https://stackoverflow.com/questions/7577410/boost-python-select-between-overloaded-methods